web/themes/joinup/templates/content/node--event--full.html.twig

```diff
@@ -10,15 +10,19 @@
 <article{{ attributes }}>
   <div class="page__content">
     {{ content.body }}
-    {{ content.field_event_spatial_coverage }}
+    {% if content.field_event_spatial_coverage|render %}
+      <div class="fieldset">
```

Comment:
Fieldset does not seem to be the right name for a solution which just works with text areas.